new_platforms/src/enc/optee/cyres_optee.c

```diff
@@ -97,9 +66,18 @@ static TEE_Result get_pta_allocated_buf(
         oe_result = OE_OUT_OF_MEMORY;
         goto done;
     }
-    memset(local_buf, 0, local_buf_size);
```

Comment:
Why do we not need this?

Reply:
PTA_CYRES returns a null-terminated string.